consul/README.md

```diff
@@ -3,28 +3,43 @@
 > _This is a work in progress, and not complete yet. Please hold off on using
 > this guide until this message disappears_
 
-There are two ways to use Consul + Spinnaker. The first assumes you don't have
-Consul running in an existing deployment, and the second assumes you do. If you
-fall in the first bucket, continue reading.
+There are two ways to use [Consul](consul.io) + Spinnaker. The first assumes
+you don't have Consul running in an existing deployment, and the second assumes
+you do. If you fall in the first bucket, continue reading.
 
 ## Configuring Consul From Scratch
 
-This assumes you're not already using Consul for service discovery.
+This assumes you're not already using Consul for service discovery. This serves
+as a cursory guide, and assumes you're familiar with the fundamentals of
+Consul. If not, [try the
+tutorial](https://www.consul.io/intro/getting-started/install.html).
 
 ### Installing Consul
 
-If you're using [Consul](consul.io), Spinnaker assumes that you've taken care of 
-installing the Consul agent on each machine you want to be discoverable. The 
+If you're using Consul, Spinnaker assumes that you've taken care of
+installing the Consul agent on each machine you want to be discoverable. The
 best way to take care of this is to create a base image with Consul installed,
 and install further packages on top of that. To do so, copy the file in
-`./install/install.sh` onto a VM, run `$ sudo install.sh`, and capture the
+`./install/install.sh` onto a VM, run `$ sudo bash install.sh`, and capture the
 resulting disk as a VM. There are many ways to do this (Packer,
```

Comment:
Could also link to this page: https://cloud.google.com/compute/docs/images/create-delete-deprecate-private-images